lib/pages/timetable/service/uni_event_provider.dart

```diff
@@ -329,18 +488,22 @@ class UniEventProvider extends EventProvider<UniEventInstance>
 
   void updateClasses(ClassProvider classProvider) {
     _classProvider = classProvider;
-    _classProvider.fetchUserClassIds(uid: _authProvider.uid).then((classIds) {
-      _classIds = classIds;
-      notifyListeners();
-    });
+    _classProvider.fetchUserClassIds(uid: _authProvider.uid).then(
```

Comment:
There's some weird unrelated reformatting here and below.

Reply:
Update: fixed it in a new commit, it was a missing semicolon.

Reply:
It wasn't missing a semicolon, this code is unrelated to your PR and there shouldn't be any changes. Maybe there's something wrong with your auto-formatter. Please revert this to the original (and the reformatted things below, including the ignore: comment where for some reason you deleted a space).